lib/foreman_remote_execution/engine.rb

```diff
@@ -183,6 +183,7 @@ class Engine < ::Rails::Engine
           permission :view_template_invocations, { :template_invocations => [:show],
                                                    'api/v2/template_invocations' => [:template_invocations] }, :resource_type => 'TemplateInvocation'
           permission :create_template_invocations, {}, :resource_type => 'TemplateInvocation'
+          permission :execute_jobs_on_infrastructure_hosts, {}
```

Comment:
Shouldn't we have some resource assigned to the permission?
If we won't this will be in miscallanous, which I'm not sure users will look for it there, but no strong opinion here.

Also do we want Execution manager to have it? (also no strong opinion about that)

Reply:
What would be the right resource for this? Host? JobInvocation?

> Also do we want Execution manager to have it?

No. It was explicitly required that the manager should not have this permission.